plugins/rts54hub/fu-rts54hub-plugin.c

```diff
@@ -37,6 +37,7 @@ fu_rts54hub_plugin_constructed(GObject *obj)
 	fu_context_add_quirk_key(ctx, "Rts54TargetAddr");
 	fu_context_add_quirk_key(ctx, "Rts54I2cSpeed");
 	fu_context_add_quirk_key(ctx, "Rts54RegisterAddrLen");
+	fu_context_add_quirk_key(ctx, "Rts54BlockSize");
```

Comment:
you need to add this to the plugin README.md too

Reply:
OK